modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -1450,7 +1450,7 @@ allowed-classes=Allowed Classes
 allowed-credentials-per-user=Allowed Credentials Per User
 allowed-credentials-per-user-description=Set the number of allowed credentials per user.
 allowed-domains=Allowed Domains
-allowed-domains-help=Set the domains which the portal is allowed to redirect to. Remove all entries to allow any domain. Specifying a domain with a leading "*." allows redirects to subdomains. PORTAL_DOMAIN will be replaced with the current domain of the portal.
+allowed-domains-help=Set the domains which the portal is allowed to redirect to. Remove all entries to allow any domain. Specifying a domain with a leading "*." allows redirects to subdomains. PORTAL_DOMAINS allows redirects to virtual hosts used by the portal.
```

Comment:
changed language key from `PORTAL_DOMAINS allows redirects to all portal's virtual hosts.` to `PORTAL_DOMAINS allows redirects to virtual hosts used by the portal.`

Reply:
I think I would make it more explicit. For example
> PORTAL_DOMAINS allows redirects to all virtual hosts configured for portal instances and sites.

What do you think?